sound/soc/sof/Makefile

```diff
@@ -7,13 +7,16 @@ snd-sof-objs := core.o ops.o loader.o ipc.o pcm.o pm.o debug.o topology.o\
 
 snd-sof-pci-objs := sof-pci-dev.o
 snd-sof-acpi-objs := sof-acpi-dev.o
+snd-sof-of-objs := sof-of-dev.o
```

Comment:
this is likely the source of conflicts and should be renamed snd-sof-of to be consistent with Jaroslav's renaming of modules.

Reply:
@plbossart fixed.